src/block-management/block-manager.js

```diff
@@ -139,6 +139,26 @@ export default class BlockManager extends React.Component<PropsType, StateType>
 		this.state.dataSource.setDirty();
 	}
 
+	insertBlocksAfter( blocks: Array<Object> ) {
+		// find currently focused block
+		let focusedItemIndex = this.findDataSourceIndexForFocusedItem();
+		if ( focusedItemIndex == -1 ) {
+			// no focus yet? This is a well knwon bug on Android...
```

Comment:
let's add a ToDo comment here to fix it instead of leaving the magic number 3 with no other explanation